app/models/form_question.rb

```diff
@@ -9,4 +9,21 @@ def self.get_form_question_types
   def self.get_questions_for_form(form_type)
   	Form.where(form_name: form_type)[0].form_questions.sort_by {|e| e.order}
   end
+
+  def self.get_list_of_questions(form_type)
+    questions_list = []
+    form_questions = self.get_questions_for_form(form_type).each do |question|
+      questions_list << question.question
+    end    
+    questions_list
+  end
+
+  def self.update_order(form_id)
+    form_questions = self.get_questions_for_form(Form.find(form_id).form_name)
+    order = 1
```

Comment:
Seed doesn't have order for the questions as I can see and the admin ordering starts to order from 1.